__tests__/integration/include-scoped/index.js

```diff
@@ -0,0 +1,3 @@
+import {MDCList} from '../web_modules/@material/list.js';
```

Comment:
basically, you'll want to import a package here that doesn't actually exist in your package.json, but does exist in your node_modules tree. You can fake this in the node_modules/ directory by manually creating a dummy package that isn't referenced in your package.json, if you'd like.

Reply:
Honestly, you don't really even need it in your node_modules dir either, if we're testing that we've moved to a static parser with no concept of what's in your node_modules or package.json.

Reply:
If we had a spec test for this yes, but for the E2E test we do need it because Rollup tries to actually fetch it in the end

Reply:
But to your point, yes I should change the test to include it in `node_modules` but have it missing from `package.json` you’re right

Reply:
Updated. Also removed `package-lock.json` for good measure too. Test still works! 🎉